ci/create_codewind_index.py

```diff
@@ -4,21 +4,23 @@
 import json
 import os
 import fnmatch
-import sys
 from collections import OrderedDict
+import argparse
+from argparse import ArgumentDefaultsHelpFormatter
 
-base_dir = os.path.abspath(os.path.dirname(sys.argv[0]))
+parser = argparse.ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatter)
 
-displayNamePrefix = "Appsody"
-if len(sys.argv) > 1:
-    displayNamePrefix = sys.argv[1]
+parser.add_argument("-n", "--namePrefix", help="Display name prefix.", default="Appsody")
+parser.add_argument("-f", "--file", help="Location of yaml files.", default=os.getcwd())
 
-# directory to store assets for test or release
-assets_dir = base_dir + "/assets/"
+args = parser.parse_args()
 
-for file in os.listdir(assets_dir):
-    if fnmatch.fnmatch(file, '*index.yaml'):
-        with open(assets_dir + file, 'r') as yamlFile, open(assets_dir + os.path.splitext(file)[0] + ".json", 'wb') as jsonFile:
+displayNamePrefix = args.namePrefix
+
+yaml_dir = args.file + "/"
```

Comment:
What is user provides the argument ending with /

Reply:
If the argument is a directory, that will pass. If the argument is a file, it will give the error: "Not a directory"

Reply:
The most recent commit handles this for files now